From f87c97317012a3b96b67237925893b8ffd5f4f50 Mon Sep 17 00:00:00 2001
From: "Maxin B. John" <maxin.john@gmail.com>
Date: Tue, 19 Dec 2017 13:46:15 +0100
Subject: [PATCH] eeprog/Module.mk: Add missing dependency

Absence of this dependency caused parallel build to run into a race
and break.

Signed-off-by: Maxin B. John <maxin.john@intel.com>
Signed-off-by: Jean Delvare <jdelvare@suse.de>
---
 eeprog/Module.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/eeprog/Module.mk b/eeprog/Module.mk
index 9d36869..d215855 100644
--- a/eeprog/Module.mk
+++ b/eeprog/Module.mk
@@ -20,7 +20,7 @@ EEPROG_TARGETS	:= eeprog
 # Programs
 #
 
-$(EEPROG_DIR)/eeprog: $(EEPROG_DIR)/eeprog.o $(EEPROG_DIR)/24cXX.o
+$(EEPROG_DIR)/eeprog: $(EEPROG_DIR)/eeprog.o $(EEPROG_DIR)/24cXX.o $(LIB_DEPS)
 	$(CC) $(LDFLAGS) -o $@ $^ $(EEPROG_LDFLAGS)
 
 #
-- 
2.7.4

